src/main/java/edu/escuelaing/ieti/matchmaking/exception/EntityNotFoundException.java

```diff
@@ -0,0 +1,31 @@
+package edu.escuelaing.ieti.matchmaking.exception;
+
+import org.springframework.util.StringUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.IntStream;
+
+public class EntityNotFoundException extends Exception {
```

Comment:
Consider extend from `RuntimeException` instead of `Exception` to avoid adding this exception in the throws clause of every method